packages/docs/docs/bots/bots-in-production.md

```diff
@@ -94,11 +94,34 @@ ls dist
 
 ```
 
-## Linking your code to a Bot Resource
+## Creating your Bot 
 
-The next step will be to create a [`Bot` resource](/docs/api/fhir/medplum/bot) using the Medplum App and link your code to the resource.
+Next step is to create the bot. 
 
-First, we'll have to create the resource in the Medplum App. If you haven't already done so, follow the instructions on the [Bot Basics tutorial](./bot-basics/#creating-a-bot) to create a new Bot resource.
+Navigate to the [Project Admin panel](https://app.medplum.com/admin/project) and copy the ID of your project. That will be your `project-id`. 
+
+Taking the `source-file` we just created at `src/my-first-bot.ts`, we will use the `create-bot` command. In our example 
+
+```bash
+npx medplum create-bot <bot-name> <project-id> <source-file> <dist-file>
+```
+
+Running this command does the following:
+
+1. Creates the Bot resource 
+2. Creates a ProjectMembership resource that connects it to a project
+3. Saves the bot to the associated project in the Medplum database
+4. Adds a bot entry to the `medplum.config.json` file in the `bots` array
+
+:::caution Note
+If you see an error, try running the command again. If it fails after 3 tries, please [**submit a bug report**](https://github.com/medplum/medplum/issues/new) or [**contact us on Discord**](https://discord.gg/UBAWwvrVeN)
+:::
+
+## Linking your code to a Bot Resource (Skip ahead if you created your bot on the CLI)
+
+If you prefer to create the bot on the Medplum App, you can follow the instructions on the [Bot Basics tutorial](./bot-basics/#creating-a-bot) to create a new Bot resource.
+
+Then the next step will be to create a [`Bot` resource](/docs/api/fhir/medplum/bot) using the Medplum App and link your code to the resource.
 
 Next, we need to tell the [Medplum CLI](https://github.com/medplum/medplum/tree/main/packages/cli) which `.ts` and `.js` files are associated with the resource by editing the `medplum.config.json` field.
```

Comment:
Since the CLI already added the entry to the `medplum.config.json`, these instructions aren't correct anymore. Instead of telling the user to add an entry to the file, maybe just ask them to open and review.